services/harmony/app/server.ts

```diff
@@ -20,19 +20,28 @@ import * as exampleBackend from '../example/http-backend';
 import cmrCollectionReader from './middleware/cmr-collection-reader';
 import * as fs from 'fs';
 
+// redact specific values from a request log
+export const requestFilter: expressWinston.RequestFilter = (req, propName) => {
```

Comment:
nit: should probably use a named function here instead of an arrow function for consistency.